dienst2/auth.py

```diff
@@ -0,0 +1,63 @@
+from django.contrib.auth.models import Group
+from django.core.exceptions import ImproperlyConfigured, SuspiciousOperation
+from mozilla_django_oidc.auth import OIDCAuthenticationBackend
+
+
+class CHConnect(OIDCAuthenticationBackend):
+    def verify_claims(self, claims):
+        verified = super(CHConnect, self).verify_claims(claims)
+        has_access = 'dienst2' in claims.get('ldap_groups', [])
```

Comment:
Maybe add the group to the settings, so for development people can use another group?

Reply:
I was thinking about skipping this check entirely in `DEBUG`, or optionally removing all authentication with a config flag? I’d prefer not to have the OIDC client id and secret in the repo, but also not require everyone to register their own client when they want to develop.

Reply:
At least moved the group for settings. We could apply my other suggestions in another PR.

Reply:
I was thinking about putting a comment on the added id and secret, so I agree with you there. 

However, I think it would be a lot easier to at least have a authenticated user even in development so the stack is not that different. Maybe we should readd the admin user and when `DEBUG = True` redirect to a login page (that is only added to the urls when `DEBUG`)

Reply:
Does Django have a default login view? I'd be nice not to have our own login view just in case somebody wants to develop without OIDC secret.